Readme.md

```diff
@@ -84,6 +78,18 @@ To contribute to PLCrashReporter, you need the tools mentioned in 1.3.3.1 above
 
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
+### 2.3 Checking out the repository
+
+PLCrashReporter has a dependency on [Protocol Buffers implementation in C](https://github.com/protobuf-c/protobuf-c) as a git submodule. Use below command to clone PLCrashReporter repository or update the repository if you have already cloned it.
+
+```
+git clone --recursive https://github.com/microsoft/PLCrashReporter-Fork.git
+```
+
+```
```

Comment:
I think those can be merged

Reply:
No. It's two different cases. Copied from App Center